### ISSUE TITLE
Built-in photo viewer (F3) doesn't respect color profiles on macOS

### ISSUE THREAD

**Issue by Andreas0602** [mucommander/mucommander#424]
The built-in photo viewer (F3) doesn't respect color profiles on macOS ==> colors are displayed wrong.

**Comment by ahadas** [comment]
@Andreas0602 can you please provide more details?

**Comment by Andreas0602** [comment]
On my iMac, I use the following color profile for my monitor:

![image](https://user-images.githubusercontent.com/79788982/111885996-67eec600-89cb-11eb-8543-d71039762a9b.png)

The problem occurs when watching a photo which has the color profile sRGB IEC61966-2.1.

The problem becomes visible when opening that photo twice: Once with Preview.app from Apple, and once with muCommander when you select the file and press F3. When you move both windows side-by-side, you can see that the colors of the photo look different. That's because Preview.app from Apple does respect color profiles, and the photo viewer of muCommander does not.

**Comment by ahadas** [comment]
@Andreas0602 seems like https://bugs.openjdk.java.net/browse/JDK-8168074, right?

**Comment by Andreas0602** [comment]
Yes, it seems to be that issue. It was already reported 2016, so there's probably not much hope that they will fix it anytime soon. Therefore, it would be great if some workaround could be implemented in muCommander.

**Comment by pskowronek** [comment]
Can you re-check with java 20? Fingers crossed.
